doc/guides/writing-tests.md

```diff
@@ -18,6 +18,15 @@ Add tests when:
 - Fixing regressions and bugs.
 - Expanding test coverage.
 
+## Test directory structure
+See the test [directory structure overview](https://github.com/nodejs/node/blob/master/test/README.md)
```

Comment:
Since this is longer than 80 characters, please move the reference down to the end of the file and only keep `[directory structure overview][]`.

Reply:
changes made.